cmd/checkpoint/main.go

```diff
@@ -311,6 +334,28 @@ func getLocalParentPods() (map[string]*v1.Pod, error) {
 	return podListToParentPods(&podList), nil
 }
 
+// getLocalRunningPods uses the /runningpods/ kubelet api to retrieve the local container runtime pod state
+func getLocalRunningPods() (map[string]*v1.Pod, error) {
+	// TODO(aaron): The kubelet api is currently secured by a self-signed cert. We should update this to actually verify at some point
```

Comment:
The checkpointer doesn't checkpoint its own secrets? :)

Yeah, it's not impossible to work around, but by relying on the kubelet API when the API server's down means we can't use that feature. More of a mental note for me when we try to enable kubelet auth.